handbook/business-operations/README.md

```diff
@@ -265,6 +265,13 @@ Within 60 days of the end of the year, follow these steps:
 - Afterward, post in #random letting folks know that the quarterly tool reconciliation and seat clearing is complete, and that any members who lost access to anything they still need can submit a ZenHub issue to BizOps to have their access restored.
 - The goal is to build deep, integrated knowledge of tool usage across Fleet and cut costs whenever possible. It will also force conversations on redundancies and decisions that aren't helping the business that otherwise might not be looked at a second time.  
 
+### Process a new vendor invoice
+- After making sure that an invoice received from a new vendor is valid, add the new vendor to the recurring expenses section of [The Numbers](https://docs.google.com/spreadsheets/d/1X-brkmUK7_Rgp7aq42drNcUg8ZipzEiS153uKZSabWc/edit#gid=2112277278) before paying the invoice.
+
+### Process a request to cancel a vendor
+- Make the cancellation notification in accordance with the contract terms between Fleet and the vendor.
```

Comment:
@hollidayn Is the cancelation notice recorded anywhere? Just want to make sure we capture all the steps

Reply:
Other than when we send it (usually email), no, we don't typically record it anywhere else.